docs/book/src/cronjob-tutorial/testdata/project/controllers/cronjob_controller_test.go

```diff
@@ -0,0 +1,217 @@
+/*
```

Comment:
It might not be part of this PR, however, IMO would be nice we add a new block of tests in the Travis to verify the book testdata and then, in this call this test implemented here to be sure that all is fine.

Reply:
I'd be happy to take a look into this :)